client/blocks/plugins-scheduled-updates-multisite/schedule-form.tsx

```diff
@@ -1,16 +1,27 @@
 import { Flex, FlexItem } from '@wordpress/components';
 import { useState, useEffect } from 'react';
+import { useSitesPluginsQuery } from 'calypso/data/plugins/use-sites-plugins-query';
 import { useSiteExcerptsSorted } from 'calypso/data/sites/use-site-excerpts-sorted';
 import { ScheduleFormFrequency } from '../plugins-scheduled-updates/schedule-form-frequency';
 import { ScheduleFormPlugins } from '../plugins-scheduled-updates/schedule-form-plugins';
-import { validateSites } from '../plugins-scheduled-updates/schedule-form.helper';
+import { validateSites, validatePlugins } from '../plugins-scheduled-updates/schedule-form.helper';
 import { ScheduleFormSites } from './schedule-form-sites';
 
 export const ScheduleForm = () => {
 	const sites = useSiteExcerptsSorted();
-	const atomicSites = sites.filter( ( site ) => site.is_wpcom_atomic );
+	const atomicSites = sites
+		.filter( ( site ) => site.is_wpcom_atomic )
+		.sort( ( a, b ) => ( ( a.name || '' ) > ( b.name || '' ) ? 1 : -1 ) );
+	const atomicSiteIds = sites.map( ( site ) => site.ID );
+
+	const {
+		data: plugins = [],
+		isLoading: isPluginsFetching,
+		isFetched: isPluginsFetched,
+	} = useSitesPluginsQuery( atomicSiteIds );
```

Comment:
Instead of showing all plugins for all sites, should we pass in `selectedSites` instead, so it only shows plugins of selected sites? (Should show all when no sites are selected)

Reply:
Based on our chat. I have updated the logic to show **the union of plugins** for selected sites.

The PR is ready for re-review.
@ouikhuan, thanks for the feedback.